<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayWindow</class>
 <widget class="QMainWindow" name="PlayWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>821</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" rowspan="2">
     <widget class="QVTKWidget" name="qvtkWidget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>846</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget_Kinect">
   <property name="accessibleName">
    <string>Kinect2</string>
   </property>
   <property name="windowTitle">
    <string>Kinect</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="6" column="1">
      <widget class="QPushButton" name="pushButton_Kin_Stop">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_Kin_Open">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="pushButton_Kin_Play">
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_nPoint">
       <property name="text">
        <string>NbPoints</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_Kin_Grab">
       <property name="text">
        <string>Grab</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QCheckBox" name="checkBox_Ada">
       <property name="text">
        <string>Use Adafruit</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_Kin_Close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_fps">
       <property name="text">
        <string>FPS</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QCheckBox" name="checkBox_ShowPC">
       <property name="text">
        <string>Show PC</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_time">
       <property name="text">
        <string>Time</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QCheckBox" name="checkBox_SavePC">
       <property name="text">
        <string>SavePC</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_Visualizer">
   <property name="windowTitle">
    <string>Visualizer</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_CleanViewer">
       <property name="text">
        <string>CleanViewer</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_AdaFruit">
   <property name="windowTitle">
    <string extracomment="BNO055">Adafruit</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_Adafruit">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="1" column="1">
      <widget class="QGroupBox" name="groupBox_Quaternion">
       <property name="title">
        <string>Quat/Cal</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="2" column="1">
         <widget class="QLabel" name="label_X">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_Y">
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_W">
          <property name="text">
           <string>W</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_Z">
          <property name="text">
           <string>Z</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QPushButton" name="pushButton_Cal">
          <property name="text">
           <string>Get Cal</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLabel" name="label_sys">
          <property name="text">
           <string>Calibration</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_Quat">
          <property name="text">
           <string>Get Quat</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QCheckBox" name="checkBox_Save">
          <property name="text">
           <string>Save?</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="2">
         <widget class="QLineEdit" name="lineEdit_Path">
          <property name="text">
           <string>/home/silvio/Desktop/2015-11-11-TestNexCave/</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_StandardMessage">
       <property name="title">
        <string>Standard Message</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="0">
         <widget class="QLabel" name="std_answer">
          <property name="text">
           <string>answer</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="std_question"/>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton_StartLoop">
          <property name="text">
           <string>StartLoop</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="pushButton_StopLoop">
          <property name="text">
           <string>StopLoop</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="2">
         <widget class="QPushButton" name="std_send">
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0" rowspan="2">
      <widget class="QGroupBox" name="groupBox_Connection">
       <property name="title">
        <string>Connection</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="pushButton_Open">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Close">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>(0) *help?
(1) *info?
(2) *baud[:{&lt;rate&gt;|list}]?
(3) *mode[:{list|text|bin}]?
(4) *endian?
(5) *reset?
(6) *status?
(7) *unit[:{list | what | what:set}]?
(8) *accel?
(9) *euler?
(A) *grav?
(B) *gyro?
(C) *mag?
(D) *quat?
(E) *temp[:A|G|M]?
(F) *fw?
(G) *echo[:{on|off}]?
(I) *cal?</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
   <container>1</container>
   <slots>
    <slot>setEnable(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>std_question</sender>
   <signal>returnPressed()</signal>
   <receiver>std_send</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>536</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>536</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
